extension/src/experiments/studio.ts

```diff
@@ -40,12 +43,16 @@ export class Studio extends Disposable {
     return `${this.baseUrl}?showOnlySelected=1&experimentReferences=${sha}&activeExperimentReferences=${sha}%3Aprimary`
   }
 
-  private async getBaseUrl(studioToken: string, gitRemoteUrl: string) {
+  private async getBaseUrl(
+    studioToken: string,
+    gitRemoteUrl: string,
+    remoteExpRefs: string[]
+  ) {
     try {
       const response = await fetch(`${STUDIO_URL}/webhook/dvc`, {
```

Comment:
Do you have plans to support `DVC_STUDIO_URL` envvar or `studio.url` config?

Reply:
Only when a user specifically asks for that functionality.

Reply:
It’s useful for testing with local Studio servers.